clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -3583,10 +3608,10 @@ void ItaniumRTTIBuilder::BuildVTablePointer(const Type *Ty) {
         llvm::ConstantExpr::getInBoundsGetElementPtr(CGM.Int8Ty, VTable, Eight);
```

Comment:
Is relative layout explicitly unsupported ? (the bitcast is incorrect)

Reply:
`llvm::ConstantExpr::getBitCast(VTable, CGM.Int8PtrTy);` is potentially doing a bitcast from address space `1` to `0` which is illegal (needs to be an `AddrSpaceCast`), the best is probably to use `VTableTy`.

Reply:
Fixed 34b17dc